### PR TITLE
[Sample]Enable automatic runtime sample

### PR DESCRIPTION
**Pull request by D-W-**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/flows/chat/chat-with-pdf/README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ fi
 
 ### CLI Example
 
+#### Run flow in local
```
Comment (wangchao1230) [inline examples/flows/chat/chat-with-pdf/README.md:34]: avoid the word local

File: examples/tutorials/run-management/cloud-run-management.ipynb [hunk 2]
```diff
@@ -329,7 +376,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.17"
+   "version": "3.8.13"
```
Comment (crazygao) [inline examples/tutorials/run-management/cloud-run-management.ipynb:379]: why this change

File: examples/flows/standard/web-classification/README.md [hunk 3]
```diff
@@ -96,15 +96,15 @@ az account set -s <your_subscription_id>
 az configure --defaults group=<your_resource_group_name> workspace=<your_workspace_name>
 
 # create run
-pfazure run create --flow . --data ./data.jsonl --stream --runtime demo-mir --subscription <your_subscription_id> -g <your_resource_group_name> -w <your_workspace_name>
-# pfazure run create --flow . --data ./data.jsonl --stream # serverless compute
+pfazure run create --flow . --data ./data.jsonl --stream --runtime demo-mir
+# pfazure run create --flow . --data ./data.jsonl --stream # automatic runtime
```
Comment (crazygao) [inline examples/flows/standard/web-classification/README.md:100]: enable this line if automatic runtime is ready.

Comment (crazygao) [reply examples/flows/standard/web-classification/README.md:100]: same with below

File: examples/tutorials/run-management/cloud-run-management.ipynb (hunk 2 above)
Comment (D-W-) [reply examples/tutorials/run-management/cloud-run-management.ipynb]: Reverted

**Review by 0mza987 — APPROVED** [review]

**Review by crazygao — APPROVED** [review]
